joomla/joomla-cms
# [5.3] Add option for show/hide title in readmore

Pull Request for Issue #34609 .

### Summary of Changes
Adding an option to the newsflash module to show, hide use global setting for the text at the read more button


### Testing Instructions
1) Read the issue and confirm the issue on Joomla 5.2 with sample data
2) apply patch
3) go to the "Latest Posts" module
4) change the "Read More with Title" options in the module and check if the hide/show have the effect we want
5) change the "Read More with Title" options in the module to "Use global article settings"
6) go to content -> options and change under articles the Read More with Title and check if the hide/show have the effect we want

### Actual result BEFORE applying this Pull Request
See issue


### Expected result AFTER applying this Pull Request
You can control from the newflash module what text is at the read more button in the newsflash module


### Link to documentations
Please select:
